README.md

```diff
@@ -132,6 +132,8 @@ Other useful settings are:
     mirror of the LineageOS source (> 200 GB)
  * `CRONTAB_TIME (now)`: instead of building immediately and exit, build at the
     specified time (uses standard cron format)
+ * `REPO_SYNC (true)`: change this to `false` if you want to skip syncing repo
+    and build the ROM as is.
```

Comment:
I think this could be explained a bit better. Instead of "and build the ROM as is", you could write "and build from the current state of the source tree."

Maybe you could also add one or two examples for when someone would like to skip repo sync.

Reply:
Note that some paths are always reset by the build scripts, see https://github.com/lineageos4microg/docker-lineage-cicd/blob/36905b464b5880945e1025c2722cac4cf45a2ab8/src/build.sh#L128-L135

This might also be worth noting here

Reply:
Thank you @bananer for pointing that out!
Added conditions for `git` commands and Updated README.